### PR TITLE
Show commercial users by default

### PR DESCRIPTION
**Pull request by victoria34**

### CODE REVIEW

File: src/main/webapp/app/pages/userManagement/UserDetailsPage.tsx [hunk 1]
```diff
@@ -46,15 +56,46 @@ export default class UserDetailsPage extends React.Component<{
...
       notifyError(e, 'Error fetching users');
     }
   }
 
   @action
-  toggleFilter(query: any) {
-    this.currentSelectedFilter = query;
+  toggleFilter(button: string) {
+    this.currentSelectedButton = button;
+    switch (this.currentSelectedButton) {
```
Comment (zhx828) [inline src/main/webapp/app/pages/userManagement/UserDetailsPage.tsx]: Do we need the switch? Seems like a if-else condition

Comment (victoria34) [reply src/main/webapp/app/pages/userManagement/UserDetailsPage.tsx]: I think it's clear and easy for extending filters in the future and switch is also good for if-else conditions more than 3.

File: src/main/webapp/app/pages/userManagement/UserDetailsPage.tsx [hunk 2]
```diff
@@ -85,7 +123,7 @@ export default class UserDetailsPage extends React.Component<{
   }
 
   private getStatus(activated: boolean) {
```
Comment (zhx828) [inline src/main/webapp/app/pages/userManagement/UserDetailsPage.tsx]: This status func is dup from user management. I didn't compare these two files, there might be more. Let's don't worry about that yet until we figure out a plan for these two pages

File: src/main/webapp/app/pages/userManagement/UserDetailsPage.tsx (hunk 1 above)
Comment (zhx828) [reply src/main/webapp/app/pages/userManagement/UserDetailsPage.tsx]: I would not worry about the extension when the requirement is not clear yet. At this point they are dup code.

**Pull request merged**